datadog_checks_base/agent_requirements.in

```diff
@@ -7,13 +7,13 @@ cryptography==2.2.2
 cx_oracle==6.0.1
 dnspython==1.12.0
 enum34==1.1.6
-gearman==2.0.2
+gearman==2.0.2; sys_platform != 'win32' and python_version < '3.0'
```

Comment:
do we need the version limit?

Reply:
Its not a hard requirement but there are only two dependencies that require Python 2, so its more serves as documentation.